pennylane/transforms/tape_expand.py

```diff
@@ -180,3 +187,81 @@ def expand_fn(tape, _depth=depth, **kwargs):
     stop_at=not_tape | is_measurement | (~is_trainable) | has_grad_method,
     docstring=_expand_invalid_trainable_doc,
 )
+
+
+@contextlib.contextmanager
+def _custom_decomp_context(custom_decomps):
+    """A context manager for applying custom decompositions. Used by the
+    create_custom_decomp_expand_fn method."""
```

Comment:
I wonder if the second sentence is important? If we delete or rename the method, or use it in another method, this becomes obsolete.